api/mongo_helper.py

```diff
@@ -62,7 +63,8 @@ def _increment_id_short_counter(self):
     @staticmethod
     def _get_connection_string(config: MongoConfig):
         return f"mongodb+srv://{config.MONGO_USERNAME}:"\
-        + f"{config.MONGO_PASSWORD}@cluster0.vwqib.mongodb.net/"\
+        + f"{config.MONGO_PASSWORD}@"\
+        + f"{config.MONGO_CLUSTER_NAME}.vwqib.mongodb.net/"\
```

Comment:
Ah, maybe not clear but I also meant to extract the rest of this host name, e.g. `cluster0.vwqib.mongodb.net` as this will vary by user/platform etc. 
e.g. for me it's cluster0.fyfgb.mongodb.net, but ideally a user would be able to also use this for other mongo servers (than just ones explicitly hosted by mongodb.net)